src/components/SidebarTabs/SharingSidebarTab.vue

```diff
@@ -60,6 +60,12 @@
 						</template>
 						{{ t('forms', 'Copy to clipboard') }}
 					</NcActionLink>
+					<NcActionButton @click.prevent="openQrModal(getPublicShareLink(share))">
```

Comment:
With the ActionButton you can use the @click event and don't have to prevent the default action.
```suggestion
					<NcActionButton @click="openQrModal(getPublicShareLink(share))">
```